Stickers/Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>2.0</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>$(CURRENT_PROJECT_VERSION)</string>
```

Comment:
Does this actually work?

Reply:
It's used here as well: https://github.com/HackIllinois/iOS/blob/dev/HIAPI/Info.plist#L20